docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -234,6 +239,14 @@ log-file: "teku_2020-01-01.log"
 
 Relative or absolute location, and filename of the log file.
 
+The default directory is OS dependant:
+
+* macOS: `~/Library/teku/log`
+* Unix/Linux: `~/.teku/log`
+* Windows: `%localappdata%\teku\log`
```

Comment:
ok so here, if the logs config variables aren't set, then the logs go to '/logs' under the data path, otherwise they'll go to the full path specified by the logs configuration (they're full path including filename). 
I'm 99% sure if you actually specify 'teku.log' it'll be teku.log in the current working folder.

Reply:
Done